packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -1067,3 +1104,78 @@ function appModuleVersion(mod: ReleasedAppModuleFragment): Required<AppModuleVer
     },
   }
 }
+
+interface AppLogsSubscribeQueryVariables {
```

Comment:
I believe once the core PR is merged, we can import these types. Leaving here for now.